src/main/resources/db/migration/local/V1_243_2__add_external_test_security_groups.sql

```diff
@@ -0,0 +1,49 @@
+UPDATE darts.user_account
```

Comment:
I'm confused with what we have locally
`select * from user_account where is_system_user is false;`

It looks like we should be setting this flag for all these users (usr_id):
```
0
-1
-2
-3
-4
-40
-41
-42
-43
```
Please can you check this?

Reply:
I set these specific system users and changed the OID. This was a change I did on staging with one of the testers. I figured it would be needed on our local db's. I didn't check if the others should also be set as system user.